cmd/operator/main.go

```diff
@@ -169,8 +173,11 @@ func parseFlags(fs *flag.FlagSet) {
 	fs.Var(&cfg.ThanosRulerSelector, "thanos-ruler-instance-selector", "Label selector to filter ThanosRuler Custom Resources to watch.")
 	fs.Var(&cfg.SecretListWatchSelector, "secret-field-selector", "Field selector to filter Secrets to watch")
 
+	// Auto GOMEMLIMIT Ratio
+	fs.Float64Var(&memlimitRatio, "auto-gomemlimit-ratio", 0.9, "The ratio of reserved GOMEMLIMIT memory to the detected maximum container or system memory. The value should be less than 0.0 setting to 0.0 and greater than 1.0 setting to 1.0. Default: 0.9")
+
 	featureGates = k8sflag.NewMapStringBool(ptr.To(make(map[string]bool)))
-	fs.Var(featureGates, "feature-gates", "Feature gates are a set of key=value pairs that describe Prometheus-Operator features. At the moment there are no feature gates available.")
+	fs.Var(featureGates, "feature-gates", "Feature gates are a set of key=value pairs that describe Prometheus-Operator features. Valid options: auto-gomemlimit.")
```

Comment:
The comment line below shows how we intend to show available feature gates.

What we want to do is add default values [here](https://github.com/prometheus-operator/prometheus-operator/blob/4c1706c4cf97c9e2db07635b3e517c1e0a949ec7/pkg/operator/feature_gates.go#L25-L26), then uncomment the line below (line 181 in this PR) and delete line 180.

Reply:
Thanks. Got it.  PTAL